lib/index.ts

```diff
@@ -24,10 +24,10 @@ export function normalizeTarEntry(name: string): string {
  * @param size
  */
 export function streamToBuffer(stream: NodeJS.ReadableStream): Promise<Buffer> {
-	return new Promise<Buffer>((resolve, reject) => {
-		let buffer: Buffer = new Buffer('');
-		stream.on('data', (data: Buffer) => buffer = Buffer.concat([buffer, data]));
-		stream.on('end', () => resolve(buffer));
+	return new Promise<Buffer[]>((resolve, reject) => {
```

Comment:
You can drop this type now, right?

Reply:
This tells typescript what we'll return from this part of the promise, so it knows we'll call `Buffer.concat` with a `Buffer[]` specifically and therefore what `Buffer.concat` will return - I can remove the `<Buffer[]>` by virtue of `Buffer.concat` always returning a Buffer but if it was overloaded with different returns then we'd lose info